runners/direct-java/pom.xml

```diff
@@ -192,6 +192,9 @@
                      the second relocation. -->
                 <relocation>
                   <pattern>com.google.common</pattern>
+                  <excludes>
```

Comment:
Add a comment stating com.google.common is too generic and we need to exclude a path that would match the guava-testlib dependency anywhere this exclude line is added.

Also, add a comment to everywhere the guava-testlib dependency is added (including root pom.xml inside dependencyManagement section) that if relocating com.google.common with the maven shade plugin, you should be warned that this will also match classes within guava-testlib and that you should exclude have an exclude for com.google.common.**.testing